### PR TITLE
Issue: have the template point to the troubleshoot page

### PR DESCRIPTION
**Pull request by akimd**
Signed-off-by: Akim Demaille <akim.demaille@docker.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
+Please, check https://docs.docker.com/docker-for-win/troubleshoot/.
+Issues without logs and details cannot be exploited, and will be closed.
```
Comment (rn) [inline .github/ISSUE_TEMPLATE.md]: exploited? probably "debugged" or "diagnosed" is better

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+
+  - [ ] I have tried with the latest version of my channel (Stable or Edge)
+  - [ ] I have submitted Diagnostics
+  - [ ] I have included the Diagnostics ID in this Issue
+
```
Comment (rn) [inline .github/ISSUE_TEMPLATE.md]: we should probably ad a field here for the windows and docker for windows version as well

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (akimd) [reply .github/ISSUE_TEMPLATE.md]: Thanks for catching this...  This is a Frenchism :(
> ▸ exploiter une situation
> to capitalize on a situation

Let's go for debugged.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (akimd) [reply .github/ISSUE_TEMPLATE.md]: Good idea!

**Pull request merged**